### DISCUSSION TITLE
Loop, Habits, Loop Habits or Loop Habit Tracker?

### DISCUSSION THREAD

**Discussion by kaoneko** [iSoron/uhabits#1757] [General Discussion]
In the app stores it's called **Loop Habit Tracker**. In the app's description it's simply referred to as **Loop**. Once installed it identifies itself as **Habits** (launcher, main screen, app info), but in the About screen it's **Loop Habit Tracker** again. When exporting the database the file names will use **Loop Habits**, the website is at **loophabits.org** and when I first started using this app I think that's also what it was called in the app stores.

I was setting up a folder on my WebDAV storage to sync database backups to and I was confused as to what I should call it, which made me realise I've been confused about what this app's name is precisely for a long time now. In my head it's always been Loop Habits, but I keep running into all these variants on the name... would it be a good idea to change things here and there so the app is consistently referred to by the same name everywhere?

<img src="https://github.com/iSoron/uhabits/assets/26896621/5ac72cf6-b9fe-488f-9e6c-b0a106352b2a" width=25% height=25%> 
<img src="https://github.com/iSoron/uhabits/assets/26896621/30d655b9-3eb6-4706-bc84-44ea89e6ee35" width=25% height=25%> 
<img src="https://github.com/iSoron/uhabits/assets/26896621/6b52c8b3-458c-4f86-a9be-63487d3249fc" width=25% height=25%> 
<img src="https://github.com/iSoron/uhabits/assets/26896621/46953bdd-10bc-4a4d-868f-779a83c8470e" width=25% height=25%> 
<img src="https://github.com/iSoron/uhabits/assets/26896621/55ee22e6-d7a5-48ce-90e0-aceb54ac944e" width=25% height=25%> 
<img src="https://github.com/iSoron/uhabits/assets/26896621/51a0069e-f5d3-4ef4-9e97-fb41749ee3fc" width=25% height=25%> 
<img src="https://github.com/iSoron/uhabits/assets/26896621/8830bdcb-210b-476f-b4aa-eb32d9e73929" width=25% height=25%> 
<img src="https://github.com/iSoron/uhabits/assets/26896621/9c26bcc2-a015-4e66-80d5-3644945b8190" width=25% height=25%>

**Comment by exanrauzer** [comment]
Yes, it is messy.

**Comment by uok** [comment]
It would also be great if app name is not translated. I could not find it in Android launcher after installation because of different name
